governance/api.go

```diff
@@ -89,7 +89,7 @@ func (api *PublicGovernanceAPI) Vote(key string, val interface{}) (string, error
 	gMode := api.governance.ChainConfig.Governance.GovernanceMode
 	gNode := api.governance.ChainConfig.Governance.GoverningNode
 
-	if GovernanceModeMap[gMode] == params.GovernanceMode_Single && gNode != api.governance.nodeAddress {
+	if GovernanceModeMap[gMode] == params.GovernanceMode_Single && gNode != api.governance.nodeAddress.Load().(common.Address) {
```

Comment:
Can 'NodeAddress()' function be used in here? 
like "gNode != api.NodeAddress()"

Reply:
@jimni1222 Yes, but not now. In following PRs, there is a change in the function 'NodeAddress()' to use load(). I'll apply your suggestion after applying it.